public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -61,17 +61,17 @@ a#write-directive
   :marked
     Read more about [style binding](template-syntax.html#style-binding) on the [Template Syntax](template-syntax.html) page.
 
-    For a simple example, though, this will demonstrate how attribute directives work.
+    For a simple example, though, this demonstrates how attribute directives work.
```

Comment:
This aside about style binding is a mistake. FIrst, you couldn't use the style binding to set the background _when the user hovers_. Second, more importantly, this aside is distraction. 

In a related PR I delete this aside and replace it with a one line example of how the new directive WILL apply to an element.  I'll fix the resulting merge conflict.